components/Snackbar/src/ColorThemer/MDCSnackbarColorThemer.m

```diff
@@ -22,12 +22,9 @@ + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme {
   MDCSnackbarManager.snackbarMessageViewBackgroundColor =
       [colorScheme.onSurfaceColor colorWithAlphaComponent:0.8f];
   MDCSnackbarManager.messageTextColor = [colorScheme.surfaceColor colorWithAlphaComponent:0.87f];
-  UIColor *buttonTitleColor = [colorScheme.surfaceColor
-                                       colorWithAlphaComponent:0.6f];
-  [MDCSnackbarManager setButtonTitleColor:buttonTitleColor
-                                 forState:UIControlStateNormal];
-  [MDCSnackbarManager setButtonTitleColor:buttonTitleColor
-                                 forState:UIControlStateHighlighted];
+  UIColor *buttonTitleColor = [colorScheme.surfaceColor colorWithAlphaComponent:0.6f];
+  [MDCSnackbarManager setButtonTitleColor:buttonTitleColor forState:UIControlStateNormal];
+  [MDCSnackbarManager setButtonTitleColor:buttonTitleColor forState:UIControlStateHighlighted];
```

Comment:
Consider reverting this file - looks like it's just a stylistic change.

Reply:
It was a result of formatting. I can revert it.